apps/docs/src/app/platform/component-docs/platform-button/platform-button-docs.component.html

```diff
@@ -11,7 +10,8 @@
 <separator></separator>
 
 <fd-docs-section-title [id]="'Button-sizes'" [componentName]="'button'"> Button Sizes </fd-docs-section-title>
-<description> Button support two sized: compact and cozy  </description>
+<description> Button support two sized: compact and cozy. Use <code class="code-snippet">[contentDensity]="''"</code>
```

Comment:
what is the value here?

Reply:
updated with [contentDensity]="''compact"

Reply:
Updated [contentDensity]="'compact"